client/verta/verta/operations/monitoring/client.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import print_function
+import itertools
+
+from verta._tracking import _Context
```

Comment:
Relative import is preferred (and required for Python 2 in some cases iirc)
```suggestion
from ..._tracking import _Context
```